dcpy/lifecycle/ingest/transform.py

```diff
@@ -182,6 +183,29 @@ def strip_columns(
             df = df.apply(lambda x: x.str.strip() if x.dtype == "object" else x)
         return df
 
+    def coerce_column_types(
```

Comment:
Agree, it may be confusing in a case where a column has different data types under `columns` and processing steps. But seems fine to have it this way for now, indeed.